README.md

```diff
@@ -117,8 +116,7 @@ shell with:
 If you're running inside of an msys shell, however, you can run:
 
 ```sh
-$ ./configure --build=x86_64-pc-windows-msvc
-$ make && make install
+$ ./x.py build --build=x86_64-pc-windows-msvc && ./x.py dist --install
```

Comment:
I think this may not work because the second command doesn't have `--build`. Perhaps this could just recommend changing the `rust.build` value in `config.toml`?